src/test/java/wellnus/atomichabit/AtomicHabitTest.java

```diff
@@ -171,23 +171,31 @@ public void updateHabit_indexOutOfBounds_atomicHabitExceptionThrown() throws Wel
      * if the user does not input positive number for the increment
      */
     @Test
-    public void updateHabit_checkIncrementNotPositive_success() throws WellNusException {
+    public void updateHabit_checkIncrementNotPositive_printErrorMessage() throws WellNusException {
         addHabit_checkOutput_success();
-        String payload = "junit test";
         String habitIndex = "1";
-        String increment = "-10";
-        String testUpdateCommand = String.format("%s --id %s --inc %s", UPDATE_HABIT_COMMAND, habitIndex, increment)
+        String increment = "-1";
+        String error_message = "Increment with minimum of 1 is expected, no update shall be performed.";
+        String warning_separator = "!!!!!!-------!!!!!--------!!!!!!!------!!!!!---------!!!!!!!";
+        String expectedErrorOutput = warning_separator
+                + System.lineSeparator()
+                + "Error Message:"
+                + System.lineSeparator()
+                + error_message
+                + System.lineSeparator()
+                + "Note:"
+                + System.lineSeparator()
+                + System.lineSeparator()
+                + warning_separator;
```

Comment:
I am not really sure about the way I implement the testing for invalid increment. Since the `BadCommandException` is caught in the `UpdateCommand`, I decided to print the error message manually. Any suggestions for this?